quipucords/scanner/manager.py

```diff
@@ -52,29 +64,35 @@ def log_info(self):
             interval = DEFAULT_HEARTBEAT
         heartbeat = Timer(interval, self.log_info)
         current_scan_job = None
-        if self.current_task:
-            current_scan_job = self.current_task.identifier
-        logger.info('Scan Manager Stats: '
-                    'current_scan_job=%s. '
-                    'queue_length=%s.',
-                    current_scan_job, len(self.scan_queue))
+        scan_job_message = None
```

Comment:
Does this variable need to be initialized to None since it is set in either the if or the else?

Reply:
Removed.